mmcv/ops/deform_conv.py

```diff
@@ -256,10 +256,14 @@ def __init__(self,
         self.reset_parameters()
 
     def reset_parameters(self):
-        n = self.in_channels
+        # fills the weight with values according to the method
+        # described in `Delving deep into rectifiers: Surpassing human-level
+        # performance on ImageNet classification` - He, K. et al. (2015), using
+        # a uniform distribution
+        fan_in = self.in_channels // self.groups
```

Comment:
I suggest to directly use kaiming_init API from nn.init or mmcv.cnn here to avoid hardcode.

Reply:
The implementation here is a little different from the [pytorch](https://github.com/pytorch/pytorch/blob/2e6221a232d39917e2736b248c53fa85dfb8986e/torch/nn/init.py#L360)